.changelog/2098.feature.md

```diff
@@ -0,0 +1,6 @@
+node: Add automatic TLS certificate rotation support
+
+It is now possible to automatically rotate the node's TLS
+certificates every N epochs by passing the command-line flag
+`worker.registration.rotate_certs`.
+Do not use this if you use sentry nodes or IAS proxies.
```

Comment:
Why does this break those two things?

Reply:
Sentry nodes need the original node's certificate to be able to connect to it over GRPC and the validator node also needs the sentry node's certificate in order to query the sentry nodes on each re-registration.
I've talked to Tadej and Peter Us, who initially wrote the sentry node, and our conclusion was that doing automatic rotations when sentry nodes are involved is not possible the way the sentry node currently works.